docs/_bin/generate-license.py

```diff
@@ -0,0 +1,696 @@
+#!/usr/bin/env python3
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import yaml
+import json
+import os
+import sys
+from html.parser import HTMLParser
+
+
+apache_license_v2 = "\n\
```

Comment:
This is a pretty big string, should we just keep this in an external file somewhere (or do we already have it somewhere maybe) and consider just loading it from that?

Reply:
Extracted as `licenses/APACHE2`.